cloudcomposerdiff/README.md

```diff
@@ -0,0 +1,184 @@
+# A python command line tool to diff two Cloud composer environments.
+
+## Running this code
+
+### Clone repository
+
+Clone the code from GitHub to your local machine
+
+### Install locally with pip
+
+```
+pip install cloudcomposerdiff
+```
+
+### Authorizing the application using gcloud
+
+Install [gcloud](https://cloud.google.com/sdk/gcloud)
+
+Select a user which has access to both composer environments to compare.
```

Comment:
```suggestion
Select a user which has access to both Cloud Composer environments to compare.
```

Also, suggest adding the IAM roles in a prereq section then mentioning them again here

Reply:
good catch. I have added IAM roles here again as suggested.